lms/static/js/student_account/views/FormView.js

```diff
@@ -33,9 +33,11 @@
             optionalStr: gettext('(optional)'),
             submitButton: '',
             isEnterpriseEnable: false,
+            showRegisterLinks: true,
 
             initialize: function(data) {
                 this.model = data.model;
+                this.showRegisterLinks = data.showRegisterLinks || this.showRegisterLinks;
```

Comment:
@yusuf-musleh This does not work as setting `FEATURES["SHOW_REGISTRATION_LINKS"] = False` sets `data.showRegisterLinks` as `false` but this expression always takes the truthy value which is the default here.

```javascript
false || true
# returns
true 
````

Reply:
Thanks for catching this! I addressed the issue. @viadanna can you maybe take a look at this PR when you get a chance? Since Navin is on Vacation.